compiler/frontend/src/module/package.rs

```diff
@@ -59,9 +59,7 @@ impl TryFrom<&str> for PackagesPath {
     type Error = String;
 
     fn try_from(path: &str) -> Result<Self, Self::Error> {
-        expanduser(path)
-            .map_err(|it| it.to_string())
-            .and_then(|it| it.as_path().try_into())
+        PathBuf::from(tilde(path).to_string()).as_path().try_into()
```

Comment:
We can avoid the allocation behind `to_string()`, the borrowed `Path` suffices:

```suggestion
        Path::new(tilde(path).as_ref()).try_into()
```